dashboard/src/actions/kube.tsx

```diff
@@ -67,10 +68,17 @@ export type KubeAction = ActionType<typeof allActions[number]>;
 export function getResourceKinds(
   cluster: string,
 ): ThunkAction<Promise<void>, IStoreState, null, KubeAction> {
-  return async dispatch => {
+  return async (dispatch, getStore) => {
     dispatch(requestResourceKinds());
     try {
-      const groups = await Kube.getAPIGroups(cluster);
+      let groups = await Kube.getAPIGroups(cluster);
+      // Ignore APIs for operators if specified
+      const operatorsEnabled = getStore().config.featureFlags?.operators === true;
+      if (!operatorsEnabled) {
+        groups = groups.filter(
```

Comment:
When operators disabled and OLM is existing in the cluster, groups are still containing stuff related to operators.

Reply:
Didn't try to not calling `getResourceKinds`, as I assumed if it was there, it was for some reason other than operators.
I'm totally fine with giving it a try and removing it. Is everything related to resources plugin already in `master` or should I create an issue to track this later?

Reply:
I believe you already tackled this [here](https://github.com/kubeapps/kubeapps/pull/4051#pullrequestreview-848600122)?